libmysql/libmysql.c

```diff
@@ -788,13 +788,15 @@ MYSQL_FIELD *cli_list_fields(MYSQL *mysql)
 MYSQL_RES * STDCALL
 mysql_list_fields(MYSQL *mysql, const char *table, const char *wild)
 {
+  #define TABLE_NAME_SIZE 65535
```

Comment:
[nit]
We might want to call it "MAX_TABLE_NAME_SIZE", WDYT?

Reply:
Sounds great!